packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -75,35 +76,78 @@ const TABLE_STATUSES = {
   ERROR: 'error',
 };
 
+const StyledTh = styled.th`
+  background: #f1f1f1;
+  font-size: 11px;
+  font-weight: 500;
+  line-height: 13px;
+  text-align: left;
+  color: ${props => props.theme.palette.text.secondary};
+  padding: 11px 20px;
+`;
+
+const TableHeader = () => (
```

Comment:
This little custom header component fixes an alignment bug in the editable table.